src/sysdolphin/baselib/class.c

```diff
@@ -247,12 +264,12 @@ HSD_Class* _hsdClassAlloc(HSD_ClassInfo* info)
     return mem_piece;
 }
 
-int _hsdClassInit(HSD_Class*)
+int _hsdClassInit(HSD_Class* arg0)
```

Comment:
```suggestion
int _hsdClassInit(HSD_Class*)
```
If it's unused, just leave it unnamed